CMakeLists.txt

```diff
@@ -401,9 +401,9 @@ IF(LMMS_BUILD_OPENBSD)
 	FIND_PACKAGE(Sndio)
 	IF(SNDIO_FOUND)
 		SET(LMMS_HAVE_SNDIO TRUE)
-        	SET(STATUS_SNDIO "OK")
-    	ELSE()
-        	SET(STATUS_SNDIO "<not found or not supported on this platform>")
+					SET(STATUS_SNDIO "OK")
+			ELSE()
+					SET(STATUS_SNDIO "<not found or not supported on this platform>")
```

Comment:
Too many tabs here (line 404-406). If you press `view` in the top of this file you can see how it looks with the GitHub formatting. I think you might have the tabs in your editor set to a low value.